config/jobs/kubernetes/sig-arch/conformance-gate.yaml

```diff
@@ -0,0 +1,16 @@
+periodics:
+- interval: 3h
+  name: apisnoop-conformance-gate
+  annotations:
+    testgrid-dashboards: sig-arch-conformance
+    testgrid-tab-name: apisnoop-conformance-gate
+    test-grid-alert-email: apisnoop@ii.coop
```

Comment:
Change to newly created group: kubernetes-sig-arch-conformance-test-failures@googlegroups.com

Reply:
Ah, thank you for that!  The email has been updated.